core/src/test/resources/applicationContext-dao.xml

```diff
@@ -48,8 +48,8 @@
 	<bean id="dbcpDataSource" destroy-method="close" class="org.apache.commons.dbcp2.BasicDataSource">
 		<property name="driverClassName" value="com.mysql.jdbc.Driver" />
 		<property name="url" value="jdbc:mysql://localhost:3306/cgds_test" />
-		<property name="username" value="${db.user}" />
-		<property name="password" value="${db.password}" />
+		<property name="username" value="cbio_user" />
+		<property name="password" value="somepassword" />
```

Comment:
Could you revert these back to `${db.user}` and `${db.password}`? This is installation specific.

Reply:
no problem. accidentally pushed back.

Reply:
@pieterlukasse , @sandertan , are we planning to support multiple species or just multiple genomes per portal instance?

Reply:
```
INSERT INTO `reference_genome` 
VALUES (1, 'human', 'hg19', 'GRCh37', NULL, 'http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips', '2009-02-01');
``` 
will add hg19 to the reference genome table, you will also need to use the importGeneData script to populate the reference_genome_gene table for your seed database. Please reference  Updating-gene-and-gene_alias-tables.md included in this PR.